i18n/language/language.go

```diff
@@ -17,23 +19,35 @@ type Language struct {
 	*PluralSpec
 }
 
+// NewLanguage returns new Language with specified tag and spec.
+func NewLanguage(tag string, spec *PluralSpec) *Language {
```

Comment:
The call sites are not specifying field names. You can either explicitly specify field names 
 like this `&Language{Tag: tag, PluralSpec: spec}` or move the declaration of `matchingTags` to the bottom of the struct.

```
type Language struct {
  	// Tag uniquely identifies the language as defined by RFC 5646.
  	//
  	// Most language tags are a two character language code (ISO 639-1)
 	// optionally followed by a dash and a two character country code (ISO 3166-1).
 	// (e.g. en, pt-br)
 	Tag string

  	*PluralSpec

	matchingTags []string
  }
```

I think the latter would be ideal because it is more backwards compatible. Also, please add doc for `matchingTags`.